tssc/step_implementers/package/maven.py

```diff
@@ -176,6 +180,22 @@ def required_runtime_step_config_keys():
         """
         return REQUIRED_CONFIG_KEYS
 
+    def _settings_file(self):
```

Comment:
@ppremru @akim309 suggest calling this `__generate_maven_settings` for consistancy.

Reply:
updated